test/unit/org/apache/cassandra/index/sai/utils/SAIRandomizedTester.java

```diff
@@ -149,6 +159,11 @@ public static double randomDoubleBetween(double min, double max)
         return min == max ? min : min + (max - min) * getRandom().nextDouble();
     }
 
+    public static double randomDouble()
+    {
+        return getRandom().nextDouble();
+    }
+
```

Comment:
Looking at these three methods, it seems like `nextDouble()` and `randomDouble()` produce the same result, since `0 + (1 - 0) * getRandom().nextDouble()` == `1 * getRandom().nextDouble()` == `getRandom().nextDouble()` right?

Reply:
Totally! Let's leave `nextDouble` in and remove `randomDouble` ?